themes/lcars.yaml

```diff
@@ -587,7 +587,7 @@
     ha-card:has(ha-gauge) {
       background: var(--lcars-card-top-color)
     }
-  /* Toolbar and dashboard modifications */
+  # Toolbar and dashboard modifications
```

Comment:
Yep, comments outside of the CSS follow YAML syntax, comments inside follow CSS standards.